docs/test_candlestick_agg.md

```diff
@@ -33,22 +27,20 @@ GROUP BY ts, symbol;
 ```SQL,non-transactional,ignore-output
 INSERT INTO stocks_real_time("time","symbol","price","day_volume")
 VALUES
-('2023-01-11 17:59:57-06','AAPL',133.445,NULL),
-('2023-01-11 17:59:55-06','PFE',47.38,NULL),
-('2023-01-11 17:59:54-06','AMZN',95.225,NULL),
-('2023-01-11 17:59:52-06','AAPL',29.82,NULL);
-```
-
-```SQL,non-transactional,ignore-output
-INSERT INTO company("symbol","name")
-VALUES
-('AAPL','Apple'),
-('PFE','Pfizer'),
-('AMZN','Amazon');
+('2023-01-11 18:59:59+00','AAPL',140,20),
+('2023-01-11 18:23:58+00','AAPL',100,10),
+('2023-01-11 17:59:57+00','AAPL',133.445,NULL),
+('2023-01-11 17:59:55+00','PFE',47.38,2000),
+('2023-01-11 12:15:55+00','PFE',1,23),
+('2023-01-11 12:00:52+00','AAPL',29.82,NULL),
+('2023-01-11 11:12:12+00','PFE',47.38,14),
+('2023-01-11 11:01:50+00','AMZN',95.25,1000),
+('2023-01-11 11:01:32+00','AMZN',92,NULL),
+('2023-01-11 11:01:30+00','AMZN',75.225,NULL);
```

Comment:
I would rather see just a few interesting buckets (several points, data doesn't just go up and to the right) than many uninteresting ones, but this is good enough here.